src/test/groovy/hudson/plugins/gradle/PropertyPassingIntegrationTest.groovy

```diff
@@ -1,5 +1,6 @@
 package hudson.plugins.gradle
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
```

Comment:
Only use Findbugs on Java sources for the whole project please. Currently, the main sources are all Java and the test sources are all Groovy.